manifests/init.pp

```diff
@@ -34,7 +34,7 @@
   $group          = 'root',
   # could be a fact like $::ipaddress or explicit ip address
   $listen_address = undef,
-  $repo           = undef,
```

Comment:
Removing `$repo` variable will break existing code (support for multiple repository sources for each distribution, which is not implemented yet). Anyway this change wouldn't be backward compatible, I'd prefer to not to change this, unless it is really important.